.github/workflows/format.yml

```
@@ -3,23 +3,23 @@
-    - uses: actions/checkout@v3
-    - uses: actions/setup-python@v4
-      with:
-        python-version: '3.11'
-        cache: 'pip' # caching pip dependencies
-    - name: Install dependencies
-      run: |
-        pip install ruff==0.11.5
-    - name: Run ruff
-      run: |
-        ruff format --check corebehrt tests
+      - uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)